t/etc/redfishpower-plugsub-blades.dev

```diff
@@ -0,0 +1,54 @@
+# Variant of redfishpower-cray-r272z30.dev that covers use of plug
+# substitution and the assumption of blades.
+#
+# Notes:
+# - hypothetical blades go through Node0
+# - for easier grepping in tests, simplify paths
+specification "redfishpower-plugsub-blades" {
+	timeout 	60
+
+	script login {
+		expect "redfishpower> "
+		send "auth USER:PASS\n"
+		expect "redfishpower> "
+		send "setheader Content-Type:application/json\n"
+		expect "redfishpower> "
+		send "setplugs Node[0-15] [0-15]\n"
+		expect "redfishpower> "
+		send "setplugs Blade[0-3] 0\n"
+		expect "redfishpower> "
+		send "setstatpath redfish/Default-{{plug}}/Stat\n"
+		expect "redfishpower> "
+		send "setonpath redfish/Default-{{plug}}/Reset {\"ResetType\":\"On\"}\n"
+		expect "redfishpower> "
+		send "setoffpath redfish/Default-{{plug}}/Reset {\"ResetType\":\"ForceOff\"}\n"
+		expect "redfishpower> "
+		send "setcyclepath redfish/Default-{{plug}}/Reset {\"ResetType\":\"ForceRestart\"}\n"
```

Comment:
If `setpath` with a `<cmd>` argument is the new way, maybe new dev scripts should start using that instead of the old `set<cmd>path` form?

Reply:
`setpath` requires you to specify plugnames, so not as convenient for setting "default" paths for everything (which is all we need in this test case).

Reply:
Oh I thought it was just there for backwards compatibility.

Reply:
it's there for that too :-)